src/main/java/apoc/diff/Diff.java

```diff
@@ -0,0 +1,110 @@
+package apoc.diff;
```

Comment:
perhaps rather call it apoc.equals.*

a diff is more detailed difference computation and output

Reply:
btw. Java equality is not the same as Cypher equality

e.g. in Cypher all these are equal `(int)1 == long)1 == (byte)1 == (float)1.0 == (double)1.0`

Reply:
reworked